nvflare/edge/emulator/cifar10_et_task_processor.py

```diff
@@ -0,0 +1,9 @@
+from et_task_processor import ETTaskProcessor
+from torch.utils.data import Dataset
+from torchvision import datasets, transforms
+
+
+class CIFAR10ETTaskProcessor(ETTaskProcessor):
+    def get_dataset(self, data_path: str) -> Dataset:
+        transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])
```

Comment:
can the transform be performed on device (i.e. torchvision works on phone)?

Reply:
torchvision lib does NOT work on phone, phone need to use other code to do the equivalent thing if they want to.

These code are just for our device emulator that runs in Mac/Linux

Reply:
For the demo, I think we can do pre-normalization, so that emulator devices and the actual training on your phone will have the same input (unless you already have the same normalization on your phone)

Reply:
good point, let me be consistent across diff language